spec/system/viewing_reviewable_akismet_post_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+describe "Viewing reviewable akismet post", type: :system do
+  fab!(:admin)
+  fab!(:group)
+  fab!(:post)
+  fab!(:reviewable) { Fabricate(:reviewable_akismet_post, target: post, topic: post.topic) }
+
+  let(:refreshed_review_page) { PageObjects::Pages::RefreshedReview.new }
+
+  before do
+    SiteSetting.reviewable_old_moderator_actions = false
+    SiteSetting.reviewable_ui_refresh = group.name
+    group.add(admin)
+    sign_in(admin)
+  end
+
+  it "allows to confirm spam" do
```

Comment:
```suggestion
  it "allows the reviewer to confirm reviewable as spam" do
```

Tiny nit about the description here.